resources/config/defaults/boards/276.txt

```diff
@@ -2,7 +2,7 @@ STAGE 276
 ROW_1 Air,Wood,Air,Wood,Air,Wood
 FROW_1 false,false,false,false,false,false
 ROW_2 Air,Wood,Air,Wood,Air,Wood
-FROW_2 false,false,false,false,false,false
+FROW_3 false,false,false,false,false,false
```

Comment:
Of course I had to mess up with the revision itself, now it should be correct.

(Next time I'll be more clean, I just learned about squashing changes into a single commit)